pkgs/applications/audio/pifi/default.nix

```
@@ -0,0 +1,18 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "pifi-${version}";
+
+  version = (import ./gemset.nix).pifi.version;
+  inherit ruby;
+  # expects Gemfile, Gemfile.lock and gemset.nix in the same directory
+  gemdir = ./.;
+
+  meta = with lib; {
+    description = "MPD web client to listen to radio, written in React and Sinatra.";
```

```suggestion
    description = "MPD web client to listen to radio, written in React and Sinatra";
```

see https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes